src/main/java/server/Serialization.java

```diff
@@ -0,0 +1,70 @@
+package server;
+
+import entity.Bond;
+import entity.Date;
+
+import java.io.*;
+
+public class Serialization{
+    private FileOutputStream file;
+    private ObjectOutputStream out;
+    private FileInputStream fileIn;
+    private ObjectInputStream in;
+
+    public Serialization() throws IOException{
+        try{
+            file = new FileOutputStream("File_Data_Storing.TXT");
+            out = new ObjectOutputStream(file);
+            fileIn = new FileInputStream("File_Data_Storing.TXT");
+            in = new ObjectInputStream(fileIn);
+        }catch (IOException ex){
+            System.out.print("Serialization Class Initialization Error");
+        }
+    }
+
+    public void Serialize(Object obj) throws IOException {
```

Comment:
Add a bit more documentation for serialize and deserialize to explain the methods purpose.